src/index.html

```diff
@@ -10,7 +11,24 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form" data-qa="big">
+      <label class="search-label">
+        <input
+        class="search-form__input"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
        <input
          class="search-form__input"
          data-qa="keypress"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```
Consider fixing indents according to the code style guide